### ISSUE TITLE
Error: can't find the source pdf of the book, resorting to manual rendering

### ISSUE THREAD

**Issue by ingegneriadelweb** [FelixFrog/pdfgrabber#98]
Hello
When I try to download some books in some platforms, such as Sanoma, the script tells me this error message:

Error: can't find the source pdf of the book, resorting to manual rendering
Manual rendering not implemented yet, contact the developer!

and the book is not downloaded. 
What can I do? Is there a solution?

Thanks very much

**Comment by FelixFrog** [comment]
Either sanoma stopped shipping the full pdf in cleartext on the open web or the regex doesn't work.
Can you contact me on telegram at [t.me/fflxx](t.me/fflxx) so that I can see if I can resolve the issue?

**Comment by ingegneriadelweb** [comment]
But the problem is only with a few books: in the same account, I succeed in downloading 80% of books, so I don't think is Sanoma the problem...

**Comment by FelixFrog** [comment]
Yes so you might have some books in your account with something different that are therefore unsupported by pdfgrabber. You should contact me privately because to fix the problem I need to examine your books.

**Comment by FelixFrog** [comment]
> The latest commit (d9b681237e7a356acfacddcaf4f2136dd80135e9) should fix this error, but the rendering might not be complete. Try with this update and let me now if the output has any anomaly
As I said in issue 105. You should try too and let me know

**Comment by Malove98** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
